cmd/snap-confine/seccomp-support.c

```diff
@@ -45,6 +47,30 @@ static const char *filter_profile_dir = "/var/lib/snapd/seccomp/bpf/";
 
 typedef struct sock_filter bpf_instr;
 
+// Keep in sync with snap-seccomp/main.go
+//
+// Header of a seccomp.bin2 filter file in native byte order.
+struct __attribute__((__packed__)) sc_seccomp_file_header {
```

Comment:
We should document this in a man page or on the forum.

Reply:
Any preference @pedronis?